### PR TITLE
[edit-task] edit tasks name and description

### PR DESCRIPTION
**Pull request by duggiemitchell**
## Cute animal pic
_Because everyone likes pictures of animals._
sloth

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._
https://trello.com/c/utZda3m5

## Description
- [x] Description is in Trello card

## Steps to test/reproduce
- Go to task detail page, click gear icon in the header to edit task name and description.

## Impacted areas of website
_Provide URLs and/or a description of where to look._
- task detail page

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screenshot from 2020-03-24 15-07-50

Screenshot from 2020-03-24 15-08-20

---

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[I want to edit the name or description of a task.](https://trello.com/c/utZda3m5/67-i-want-to-edit-the-name-or-description-of-a-task)

File: src/js/components/projects/detail.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const ProjectDetail = (props: RouteComponentProps) => {
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
   const usersAssignedToTasks = new Set<string>();
-  (tasks || []).forEach((task) => {
+  (tasks || []).forEach(task => {
```
Comment (duggiemitchell) [inline src/js/components/projects/detail.tsx]: I've switched to using a workspace so I think I need to change some of the formatting settings.

Comment (jgerigmeyer) [reply src/js/components/projects/detail.tsx]: Hm... Looks like a `prettier` change -- are you sure you have the latest deps installed via `./derrick build`, or just `yarn` and `yarn lint:js` outside the container?

Comment (duggiemitchell) [reply src/js/components/projects/detail.tsx]: I _think_ so, but I'll that.

**Pull request merged**